api_app/analyzers_manager/observable_analyzers/phishstats.py

```diff
@@ -0,0 +1,56 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class PhishStats(ObservableAnalyzer):
+    """
+    Analyzer that uses PhishStats API to check if the observable is a phishing site.
+    """
+
+    base_url: str = "https://phishstats.info:2096/api/"
+
+    def __build_phishstats_url(self) -> str:
+        if self.observable_classification == self.ObservableTypes.IP:
+            endpoint = "phishing?_where=(ip,eq,{input})"
+            name = self.observable_name
+        elif self.observable_classification == self.ObservableTypes.URL:
+            endpoint = "phishing?_where=(url,like,~{input}~)&_sort=-id"
+            domain = self.observable_name.split("/")[2]
+            name = domain.split(".")[0]
+        elif self.observable_classification == self.ObservableTypes.DOMAIN:
+            endpoint = "phishing?_where=(url,like,~{input}~)&_sort=-id"
+            name = self.observable_name.split(".")[0]
```

Comment:
this is not needed, u can pass the observable_name as is to the input.

Reply:
https://phishstats.info:2096/api/phishing?_where=(url,like,~github.com~)&_sort=-date

Try this, it doesn't seem to work

Reply:
The page took 20-30 seconds to load but it does work.

Reply:
We can wait as long as it takes, all analyzers are async running in parallel. If you are getting `SoftTimeLimitExceeded` error then just increase the `soft_time_limit` value.